azure-pipelines.yml

```diff
@@ -35,9 +35,9 @@ variables:
   - name: UnitTestProjectPath
     value: 'analyzers\tests\SonarAnalyzer.Test\'
   - name: UnitTestResultsPath
-    value: '$(Agent.TempDirectory)\TestResults'
+    value: '$(Build.SourcesDirectory)\TestResults'
```

Comment:
Unrelated deconfusion, to use the same directory before and after the artifact is restored